### DISCUSSION TITLE
DOUBT: Only include selected snippet in the context

### DISCUSSION THREAD

**Discussion by RitickMadaan** [zed-industries/zed#21365] [Feature Requests]
### Check for existing issues

- [X] Completed

### Describe the feature

When using ctrl + enter on a selected code snippet, the whole buffer is included in the context which might be unnecessary in some cases . Is there a way to only include the selected snippet?

### Environment

0.164.0

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
